samples/sample_multi_transcode/include/pipeline_transcode.h

```diff
@@ -797,6 +797,7 @@ namespace TranscodingSample
 
         // HEVC
         mfxExtHEVCParam          m_ExtHEVCParam;
+        mfxExtHEVCTiles          m_ExtHEVCTiles;
```

Comment:
You need to add initialization of new member to constructor

Reply:
Thanks, I fixed it.

Reply:
I suggest to merge your two commits to one. Now there is a state in git history when this feature is in some intermediate state